pennylane/devices/legacy_facade.py

```diff
@@ -184,6 +184,14 @@ def tracker(self, new_tracker):
     def name(self) -> str:
         return self._device.short_name
 
+    def capabilities(self) -> dict:
+        """To override the Device class's capabilities property
+
+        TODO: unify the behaviour of the old `capabilities` and the new.
+
+        """
+        return self._device.capabilities()
+
```

Comment:
Unless we can map the child capabilities to a `DeviceCapabiltiies` class, we should probably just return `None` here.

Reply:
Does any legacy device still depends on its `capabilities` being a method that returns a dictionary? Many of our tests currently do, e.g., 

https://github.com/PennyLaneAI/pennylane/blob/4986a3a356c86d0b73f468eab0bd47458bba27cd/tests/measurements/test_state.py#L357

Should we update the tests to get rid of its dependency on the `capabilities` method?